### ISSUE TITLE
add `linear_pool` to vignette

### ISSUE THREAD

**Issue by elray1** [Infectious-Disease-Modeling-Hubs/hubEnsembles#29]

**Comment by nickreich** [comment]
@lshandross I see you moved this to In Progress a while back. Are you actively working on this?

**Comment by lshandross** [comment]
Not exactly. The thought was that once the manuscript was finalized, we would copy everything but the case study (and maybe modify the introduction) to the vignette. The case study has a section on using the linear pool function, but updating the vignette is on hold until the manuscript is done
